src/func-cosmosdb-worker/src/tests/Functional/xxENSONOxx.xxSTACKSxx.Worker.FunctionalTests.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Is there two sln files?

Reply:
Yes, there were!  Thanks very much.
It looks like one was left behind after moving the solution. 
Cleaned up now.